awslambda/cml/cml_manager.py

```diff
@@ -92,80 +73,84 @@ def manage_labs(self) -> tuple:
                 card_sent_date = lab_data["card_sent_date"]
                 user_responded_date = lab_data["user_responded_date"]
                 lab_discovered_date = lab_data["lab_discovered_date"]
 
                 # check see if user was sent a card and never responded in time
-                if self.lab_to_wipe(
+                if self.lab_to_stop(
                     card_sent_date,
                     user_responded_date,
                     lab_discovered_date,
-                    lab_is_wiped,
+                    lab_is_stopped,
                 ):
-                    self.logging.info("Adding lab to be wiped")
-                    labs_to_wipe.append(
-                        {"lab_id": lab_id, "reason_lab_wiped": self.reason_lab_wiped}
+                    self.logging.info("Adding lab to be stopped")
+                    labs_to_stop.append(
+                        {
+                            "lab_id": lab_id,
+                            "reason_lab_stopped": self.reason_lab_stopped,
+                        }
                     )
 
-                # check see if lab within warning wiped period
+                # check see if lab within warning stopped period
                 elif self.lab_to_warn_wiping(
-                    user_responded_date, card_sent_date, lab_is_wiped
+                    user_responded_date, card_sent_date, lab_is_stopped
                 ):
-                    self.logging.info("Adding lab to warning for being wiped")
-                    labs_warning_wiped.append((lab_id, lab_title, user_responded_date))
+                    self.logging.info("Adding lab to warning for being stopped")
+                    labs_warning_stopped.append(
+                        (lab_id, lab_title, user_responded_date)
+                    )
 
                 # check see if within deletion period
                 elif self.lab_to_delete(
-                    lab_wiped_date, user_responded_date, lab_is_wiped
+                    lab_stopped_date, user_responded_date, lab_is_stopped
                 ):
                     self.logging.info("Adding lab to be deleted")
-                    labs_to_delete.append(lab_id)
+                    labs_to_delete.append({"lab_id": lab_id, "user_email": user_email})
 
-                # check see if lab within wiped period
+                # check see if lab within stopped period
                 elif self.lab_to_warn_delete(
-                    lab_wiped_date,
+                    lab_stopped_date,
                     user_responded_date,
-                    lab_is_wiped,
+                    lab_is_stopped,
                     lab_id,
                     lab_title,
                     user_email,
                 ):
-                    self.logging.info("Adding lab to warning for being wiped")
-                    labs_warning_deleted.append((lab_title, lab_wiped_date))
+                    self.logging.info("Adding lab to warning for being stopped")
```

Comment:
Might have missed it elsewhere, but you should consider adding the lab id/name/some unique identifier to these messages so that you can correlate them back to something specific